src/main/java/teammates/client/scripts/RepairTeamNameInStudentResponseAndCommentAttributes.java

```diff
@@ -0,0 +1,186 @@
+            int numberOfCommentsWithExtraSpacesInRecipient = removeExtraSpacesInComments(allComments);
+            
+            if (isPreview) {
+                System.out.println("There are/is " + numberOfStudentsWithExtraSpacesInTeamName
+                                   + "/" + totalNumberOfStudents + " student(s) with extra spaces in team name!");
+                System.out.println("There are/is " + numberOfReponsesWithExtraSpacesInRecipient
+                                   + "/" + totalNumberOfResponses + " response(s) with extra spaces in recipient and/or giver!");
+                System.out.println("There are/is " + numberOfCommentsWithExtraSpacesInRecipient
+                                   + "/" + totalNumberOfComments + " comment(s) with extra spaces in recipient!");
+                             
+            } else {
+                System.out.println("" + numberOfStudentsWithExtraSpacesInTeamName 
+                                   + "/" + totalNumberOfStudents + " student(s) have been fixed!");
+                System.out.println("" + numberOfReponsesWithExtraSpacesInRecipient 
+                                   + "/" + totalNumberOfResponses + " response(s) have been fixed!");
+                System.out.println("" + numberOfCommentsWithExtraSpacesInRecipient 
+                                   + "/" + totalNumberOfComments + " comment(s) have been fixed!");
+                System.out.println("Extra space removing done!");
+            }
+        } catch (InvalidParametersException | EntityDoesNotExistException | EntityAlreadyExistsException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private String extractStringsWithExtraSpace(Set<String> set) {
+        StringBuilder result = new StringBuilder();
+        for (String s : set) {
+            if (hasExtraSpaces(s)) {
+                result.append(s + " ");
+            }
+        }
+        return result.toString();
+    }
+    
+    private int removeExtraSpacesInComments(List<CommentAttributes> allComments) 
+                    throws InvalidParametersException, EntityDoesNotExistException {
+        int numberOfCommentWithExtraSpacesInRecipient = 0;
+        for (CommentAttributes comment : allComments) {
+            if (hasExtraSpaces(comment.recipients)) {
+                numberOfCommentWithExtraSpacesInRecipient++;
+                if (isPreview) {
+                    String recipientsWithExtraSpace = extractStringsWithExtraSpace(comment.recipients);
+                    System.out.println("" + numberOfCommentWithExtraSpacesInRecipient 
+                                       + ". \"" + recipientsWithExtraSpace + "\""
+                                       + "courseId: " + comment.courseId);
+                } else {
+                    comment.recipients = StringHelper.removeExtraSpace(comment.recipients);
+                    commentsLogic.updateComment(comment);
+                }
+            }
+        }
```

Comment:
Instead of wrapping the whole thing inside an `if`, use a guard clause to reduce nesting. e.g. `if(!condition) continue`